.gitignore

```diff
@@ -1,4 +1,6 @@
 .vscode
+bin/
+target/
```

Comment:
In addition to adding the directory to .gitignore, please also remove the directory from the repository completely.